contrib/zenith/relsize_cache.c

```diff
@@ -33,15 +33,21 @@ typedef struct
 
 typedef struct
 {
-	RelTag tag;
+	RelTag		tag;
 	BlockNumber size;
 } RelSizeEntry;
 
 static HTAB *relsize_hash;
 static LWLockId relsize_lock;
-static int relsize_hash_size;
+static int	relsize_hash_size;
 static shmem_startup_hook_type prev_shmem_startup_hook = NULL;
 
+/*
+ * Size of cache entry is 20 bytes. So 64 entry will take about 1.2 Mb,
+ * which seems to be a reasonable default.
```

Comment:
Maybe `entry` -> `entries`, while on it?

Reply:
Yeah, and the math actually doesn't make sense sense either. Fixed.